src/main/scala/fi/oph/kouta/domain/toteutusMetadata.scala

```diff
@@ -244,20 +245,33 @@ package object toteutusMetadata {
   val models = List(Opetus, ToteutusMetadata, KorkeakouluOsaamisala, Osaamisala, KorkeakouluToteutusMetadata, AmmattikorkeaToteutusMetadata, YliopistoToteutusMetadata, AmmatillinenToteutusMetadata)
 }
 
-sealed trait ToteutusMetadata extends Validatable {
+sealed trait ToteutusMetadata extends ValidatableSubEntity {
   val tyyppi: Koulutustyyppi
```

Comment:
Olisi tietysti kiva, jos tämän voisi validoida, mutta se ei taida onnistua helposti, kun se koulutuksen koulutustyyppi löytyy vain koulutukselta....

Reply:
Tää on ehkä osa isompaa kokonaisuutta. Pitäisi varmaan tarkistaa, että mitään näitä, joilla on koulutustyyppi suoraan tai epäsuoraan ei voi liittää toisiinsa, ellei niillä ole sama tyyppi. Eli koulutus, toteutus, valintaperuste, sorakuvaus ja välillisesti hakukohde.

Reply:
Totta. Sinnehän tarvitaan myös jossain vaiheessa muutakin validointia ristiin esim. niihin tiloihin ja julkaisuun liittyen, joten tätä voisi varmaan miettiä siinä yhteydessä.